lxd/db/instances.go

```diff
@@ -106,11 +106,6 @@ type InstanceFilter struct {
 	Type    *instancetype.Type
 }
 
-// InstanceFilterAllInstances returns a predefined filter for returning all instances.
-func InstanceFilterAllInstances() *InstanceFilter {
```

Comment:
@masnax can you split the removal of this function into a separate commit, as is not associated with the commit message. Thanks